src/main/java/com/playtika/homework2/task1/helper/FindByPower.java

```diff
@@ -0,0 +1,37 @@
+package com.playtika.homework2.task1.helper;
+
+import com.playtika.homework2.task1.exceptions.NoElementInListException;
+import com.playtika.homework2.task1.model.Device;
+
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+public class FindByPower {
+
+    public static List<Device> findByPower(List<Device> allDevices) {
+        System.out.println("\n\nПоиск прибора по мощности.\n");
+        try {
+            Scanner scan = new Scanner(System.in);
+            System.out.println("Введите мощность: ");
+            int power = scan.nextInt();
+            Device values = null;
```

Comment:
you should create List<Device> and add elements to it while iterating `allDevices`